src/renderer/screens/exchange/Swap2/Form/ToRow.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import React from "react";
+import { Trans } from "react-i18next";
+import { BigNumber } from "bignumber.js";
+import Box from "~/renderer/components/Box/Box";
+import InputCurrency from "~/renderer/components/InputCurrency";
+import SelectCurrency from "~/renderer/components/SelectCurrency";
+import { amountInputContainerProps, selectRowStylesMap } from "./utils";
+import { FormLabel } from "./FormLabel";
+import { listCryptoCurrencies } from "@ledgerhq/live-common/lib/currencies";
+import type { CryptoCurrency } from "@ledgerhq/live-common/lib/types/currencies";
+
+type Props = {
+  toCurrency: ?CryptoCurrency,
+  setToCurrency: (?CryptoCurrency) => void,
+  toAmount: ?BigNumber,
+  setToAmount: BigNumber => void,
+};
+
+export default function ToRow({ toCurrency, setToCurrency, toAmount, setToAmount }: Props) {
+  // dummy
+  const currencies = listCryptoCurrencies();
+  const unit = toCurrency && toCurrency.units[0];
+
+  return (
+    <>
+      <Box horizontal mb="8px" color={"palette.text.shade40"} fontSize={3}>
```

Comment:
`mb={2}` should automatically be 10px, there's a mapping on the theme level for this, if design put 8... make it 10, closest one here. https://github.com/gre/ledger-live-desktop/blob/3d665c6db419194879fba8c13ad9be285d156c09/src/renderer/styles/theme.js#L6-L7